upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -1141,6 +1141,8 @@ func setupNetworking(opt *NewClusterOptions, cluster *api.Cluster) error {
 	case "cilium-etcd":
 		addCiliumNetwork(cluster)
 		cluster.Spec.Networking.Cilium.EtcdManaged = true
+	case "cilium-eni":
+		addCiliumNetwork(cluster)
```

Comment:
It doesn't look like this is actually enabling ENI mode.

Reply:
Heh. I missed a commit as I wanted the test diff to be more visible. Added now.